src/components/app/Home.js

```diff
@@ -355,14 +371,15 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
               href="https://github.com/devtools-html/perf.html"
               target="_blank"
               rel="noopener noreferrer"
+              title="Go to our git repository"
```

Comment:
Can you add `(open a new page)` ?

Reply:
sorry i don't understand, do you mean add this term to title

Reply:
Yes, please :) sorry for the confusion

Reply:
did you mean "Go to our git repository, open a new page"?